amida-tech/orange-api
# Feature/user avatars

# What does it do
Adds avatars to users in the same way that patients have them.

# How to test
GET new endpoint `/user/avatar` it should be the default image
POST a picture to `/user/avatar`
GET `/user/avatar` again, it should be the new picture

Use orange-ignite branch of same name and make sure nothing crashes, photo upload still works, etc.
